dcp/DcpHandler.go

```diff
@@ -525,9 +525,24 @@ func (m *Mutation) ToUprEvent() *xdcrBase.WrappedUprEvent {
 //	colFiltersLen - 2 byte (number of collection migration filters)
 //	(per col filter) - 2 byte
 func (mut *Mutation) Serialize() []byte {
+	var bodyHash [64]byte
+	var xattrSize uint32
+	var xattr []byte
+
+	if mut.Datatype&xdcrBase.XattrDataType > 0 {
```

Comment:
This change will strip away all Xattr. Is this what you intended? Or are you only supposed to take out HLV related xattr?

Reply:
I did think about it and eventually I decided to strip the Xattrs as whole and write it to file.

Reason : As of now the scope of the project is limited hlv comparison but in the future if at all we decide to compare other system Xattrs (say eventing,fts) it becomes clumsy to write/read each system Xattr independently to file.

My rationale was that by keeping the Xattrs as a whole separate, we can easily create an iterator to select the specific system Xattrs whenever needed.

However, this approach introduces a tradeoff: for body comparison, we would need to run an iterator to strip the HLVs and any other Xattrs that the user chooses to exclude from the comparison, taken as input from the command line.